src/lib/product_reviews/review/author.js

```diff
@@ -29,9 +29,7 @@ const Author = ({ author, reviewsUrl }) => {
       <Container>
          <Link href={url}>
             <Avatar src={avatar} size={28} />
-            <div itemProp="author" itemScope="" itemType="https://schema.org/Person">
-               <Name itemprop="name">{name}</Name>
-            </div>
+            <Name>{name}</Name>
```

Comment:
Just so I understand correctly, this was unsucessfully trying to put an author's name in the head, and we are now sucessfully including this information when we build the schemaReviews in `src/lib/product_reviews/visualizer/index.js`?

Reply:
This data is getting replaced by visualizer meta tag insertion yeah. 
We used to have a itemprop/meta tag setup that was broken and got replaced by ld+json so that's why this specifically was removed.